odxtools/cli/_print_utils.py

```diff
@@ -293,9 +294,10 @@ def print_dl_metrics(variants: list[DiagLayer]) -> None:
         all_services: list[DiagService | SingleEcuJob] = sorted(
             variant.services, key=lambda x: x.short_name)
         ddds = variant.diag_data_dictionary_spec
+        comparam_refs = variant.comparam_refs if isinstance(variant, HierarchyElement) else None
```

Comment:
this way avoids extra checks below 
```suggestion
        comparam_refs = variant.comparam_refs if isinstance(variant, HierarchyElement) else []
```